Andrea-Harris.txt

```diff
@@ -1 +1,104 @@
-<h1> Hello! I'm Andrea Harris</h1>touch
\ No newline at end of file
+   https://codepen.io/AsHarris/full/YzzrXbj
+   
+   
+   
+    1. What is Semantic HTML? 
+    -Hypertext Markup Language.
+    2. What is HTML used for? 
+    -To create webpages, the back end the structure you can't see in plaine sight when looking at a webpage.
+    3. What is an attribute and where do we put it?
+    - They give meaning to the a tag, we put it inside the elements opening tag. Exmple: <img src=""> 
+    4. What is the h1 tag used for? How many times should I use it on a page?
+    -For the most important title or sentence (Header). We should used it only 1 time.
+    5. Name two tags that have required attributes
+    -Url and a Checkbox.
+    6. What do we put in the head of our HTML document? 
+    -The most important title or sentence (<h1>) and yout <nav>.
```

Comment:
Here I think you mixed up head with header, this is correct for what goes in the header, but the <head> element can include a title for the document, scripts, styles, meta information, etc.